UPDATING.md

```diff
@@ -23,6 +23,8 @@ assists people when migrating to a new version.
 
 ## Next
 
+* [10887](https://github.com/apache/incubator-superset/pull/11172): Breaking change: SQL templating is turned off be default. To turn it on set `ENABLE_TEMPLATE_PROCESSING` to True on `DEFAULT_FEATURE_FLAGS`
```

Comment:
while you're in here, could you move the existing `Next` stuff to `0.37.2`? Looks like that wasn't done yet.

Also, make sure you update the PR number

Reply:
Just checked, and the UPDATING is absolutely correct, there are no warning or breaking changes on 0.37.2

Reply:
ah, gotcha. i forgot that 0.37.2 was only cherrypicked PRs